sidecar-injector/deployment/webhook-create-signed-cert.sh

```diff
@@ -46,9 +46,9 @@ while [[ $# -gt 0 ]]; do
     shift
 done
 
-[ -z ${service} ] && service=secretless-sidecar-injector-webhook-svc
-[ -z ${secret} ] && secret=secretless-sidecar-injector-webhook-certs
-[ -z ${namespace} ] && namespace=default
+[ -z ${service} ] && usage
```

Comment:
I know these are terse but it's much easier to follow the code with expanded `if` blocks